src/pages/benefits/viewAllBenefit/ApplicationDetails.tsx

```
@@ -537,6 +609,81 @@ const ApplicationDetails: React.FC = () => {
+  const EnhancedTab = ({ 
+    title, 
+    statusText, 
+    icon: StatusIcon, 
+    color,
+    tabStyles 
+  }: {
+    title: string;
+    statusText: string;
+    icon: React.ElementType;
+    color: string;
+    tabStyles?: Record<string, any>;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)